sdk/containerregistry/Azure.Containers.ContainerRegistry/src/Models/ContainerRegistryAudience.cs

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.Containers.ContainerRegistry
+{
+    /// <summary> Cloud audiences available for ACR. </summary>
+    public readonly partial struct ContainerRegistryAudience : IEquatable<ContainerRegistryAudience>
+    {
+        private readonly string _value;
+
+        /// <summary> Determines if two <see cref="ContainerRegistryAudience"/> values are the same. </summary>
+        /// <exception cref="ArgumentNullException"> <paramref name="value"/> is null. </exception>
+        public ContainerRegistryAudience(string value)
+        {
+            _value = value ?? throw new ArgumentNullException(nameof(value));
+        }
+
+        private const string AzureChinaValue = "https://management.chinacloudapi.cn";
+        private const string AzureGermanyValue = "https://management.microsoftazure.de";
+        private const string AzureGovernmentValue = "https://management.usgovcloudapi.net";
+        private const string AzurePublicCloudValue = "https://management.azure.com";
+
+        /// <summary> Azure China. </summary>
+        public static ContainerRegistryAudience AzureChina { get; } = new ContainerRegistryAudience(AzureChinaValue);
+
+        /// <summary> Azure Germany. </summary>
+        public static ContainerRegistryAudience AzureGermany { get; } = new ContainerRegistryAudience(AzureGermanyValue);
+
+        /// <summary> Azure Government. </summary>
+        public static ContainerRegistryAudience AzureGovernment { get; } = new ContainerRegistryAudience(AzureGovernmentValue);
+
+        /// <summary> Azure Public Cloud. </summary>
+        public static ContainerRegistryAudience AzurePublicCloud { get; } = new ContainerRegistryAudience(AzurePublicCloudValue);
```

Comment:
This is the only cloud where we have include 'Cloud' in the name. Should we not make this consistent across all?

Reply:
I was trying to be consistent with what we have in Azure.Identity: https://docs.microsoft.com/en-us/dotnet/api/azure.identity.azureauthorityhosts?view=azure-dotnet